examples/rust/components/keyvalue-messaging/local.wadm.yaml

```diff
@@ -36,6 +36,8 @@ spec:
               - name: wasmcloud
                 properties:
                   bucket: "WASMCLOUD"
+                  #enable_bucket_auto_create: "true"
```

Comment:
Is there a reason to not have this enabled here?

If so, would it be worth adding a comment above for why/when to enable this?

Reply:
That's a good point -- this *would* make using this example easier, not sure what people were doing before (they must have been creating the bucket manually). 


Will enable this for the local example!